.github/workflows/check_with_mesh_cli.yml

```diff
@@ -26,12 +26,18 @@ jobs:
           cd $GITHUB_WORKSPACE/cmd/rosetta && go build .
           cd $GITHUB_WORKSPACE/systemtests && go build ./proxyToObserverAdapter.go
 
-      - name: check:data
+      - name: check:construction (native)
         run: |
-          PYTHONPATH=. python3 ./systemtests/check_with_mesh_cli.py --mode=data --network=testnet
+          PYTHONPATH=. python3 ./systemtests/check_with_mesh_cli.py --mode=construction-native --network=testnet
+          sleep 30
```

Comment:
Some delay, so that the highest final block passes the block holding the last transaction of the step.